de.metas.handlingunits.client/src/main/java/de/metas/handlingunits/client/terminal/editor/model/impl/MovementsAnyWarehouseModel.java

```diff
@@ -87,4 +85,40 @@ protected void createMovements()
 
 	}
 
+	/**
+	 * Get the warehouses existing for the hus' organization , excluding those which currently contain the HUs given as parameters
+	 * 
+	 * @param hus
+	 * @return
+	 */
+	public static List<org.compiere.model.I_M_Warehouse> getWarehousesUnlessOfHUs(final List<I_M_HU> hus)
```

Comment:
teo and i had trouble understand this method from either javadoc or name. We suggest to rename it e.g. to 
`retrieveWarehousesWhichContainNoneOf`

Reply:
Thanks for the suggestion.